daac/main.tf

```diff
@@ -31,7 +31,7 @@ resource "aws_s3_bucket" "internal-bucket" {
   lifecycle {
     prevent_destroy = true
   }
-  tags = local.default_tags
```

Comment:
On reread of this - the default tag(s) would no longer be included - should this be:
```
tags = merge(local.default_tags, local.dar_yes_tags)
```
instead?  (Note: might not be syntactically right)